solvation_analysis/tests/test_plotting.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pytest
+
+from solvation_analysis.plotting import (
+    plot_network_size_histogram,
+    plot_shell_size_histogram,
+    compare_pairing,
+    compare_coordination_numbers,
+    compare_residence_times,
+    compare_speciation,
+    format_graph,
+)
+
+
+def test_plot_network_size_histogram(networking):
```

Comment:
In general there is no need to test that plotting works IMO. 
This is because these tests are not really very informative (they will always pass). 

As long as you have verified the plotting yourself I am happy with that but @orionarcher  may have a different opinion.

Reply:
I agree they aren't informative from the perspective of CI but I think they are informative for the developer. In my experience, visualization tests make it easier to use test-driven development and to iterate later on. If you are concerned about the CI time, we could comment out these lines:

```python
fig = plot_network_size_histogram(networking)
fig.show()
```

so that the tests are super cheap.

Reply:
Yeah I think commenting out fig.show is a good compromise. :)

Reply:
@orionarcher its your project so your call at the end of the day. :)